Documentation/reportdatasources.md

```diff
@@ -145,7 +145,12 @@ spec:
 [default-storage-location]: storagelocations.md#default-storagelocation
 [architecture]: metering-architecture.md
 [presto-types]: https://prestodb.io/docs/current/language/types.html
+<<<<<<< HEAD
 [reportgenerationqueries]: reportgenerationqueries.md
 [reportgenerationquery]: reportgenerationqueries.md
 [query-inputs]: reportgenerationqueries.md#query-inputs
+=======
```

Comment:
The rebase somehow included this

Reply:
That was a poor line # to place that comment on, here's the full block:
```
<<<<<<< HEAD
[reportgenerationqueries]: reportgenerationqueries.md
[reportgenerationquery]: reportgenerationqueries.md
[query-inputs]: reportgenerationqueries.md#query-inputs
=======
[reportqueries]: reportqueries.md
[query-inputs]: reportqueries.md#query-inputs
>>>>>>> *: Rename ReportGenerationQuery to ReportQuery
[prestotables]: prestotables.md
```

Reply:
Oh I probably messed that up in a different rebase.